packages/salesforcedx-vscode-core/src/commands/forceProjectCreate.ts

```diff
@@ -246,22 +290,28 @@ const parameterGatherer = new CompositeParametersGatherer(
 );
 const pathExistsChecker = new PathExistsChecker();
 
-const sfdxProjectCreateCommandlet = new SfdxCommandlet(
-  workspaceChecker,
-  parameterGatherer,
-  new ForceProjectCreateExecutor(),
-  pathExistsChecker
-);
 export async function forceSfdxProjectCreate() {
+  const createTemplateExecutor = sfdxCoreSettings.getTemplatesLibrary()
+    ? new LibraryForceProjectCreateExecutor()
+    : new ForceProjectCreateExecutor();
+  const sfdxProjectCreateCommandlet = new SfdxCommandlet(
+    workspaceChecker,
+    parameterGatherer,
+    createTemplateExecutor,
+    pathExistsChecker
+  );
   await sfdxProjectCreateCommandlet.run();
 }
 
-const projectWithManifestCreateCommandlet = new SfdxCommandlet(
-  workspaceChecker,
-  parameterGatherer,
-  new ForceProjectCreateExecutor({ isProjectWithManifest: true }),
-  pathExistsChecker
-);
 export async function forceProjectWithManifestCreate() {
+  const createTemplateExecutor = sfdxCoreSettings.getTemplatesLibrary()
```

Comment:
I'm not sure what's happening but the project create commands fails whether the flag is on or off. I think it might be something close to here.

Reply:
The flow stops/fails when it should open the file explorer window so a user can pick where they want the project to be created.

Reply:
Anything logged on the extension host output? I can create with flag on/off and with/without manifest.

Reply:
I didn't see any error on the output panel or the dev tools console. I didn't looked too close into the issue since I was going through the rest of the commands in this PR which they did work as expected.

I also did a clean install/compile of your branch so you might want to see if there's something in your cache that didn't make it into this PR. Or you can test with the vsix files built in CircleCI, I haven't test with those yet.

Reply:
Not sure what's different about our setup. I've tested the vsix file (Core extension) in https://app.circleci.com/pipelines/github/forcedotcom/salesforcedx-vscode/1874/workflows/3afc06b0-006f-45f4-84ea-e8a2beb2ff63/jobs/11661/artifacts and creating standard/empty/analytics projects are fine with/without manifest.

Reply:
I just ran some tests with the vsix files and looks like I have some issue on my local setup since everything worked properly. Thanks for checking